src/workflows/pdf/proposal/AutoProposalPdfFactory.ts

```diff
@@ -206,8 +206,8 @@ export class AutoProposalPdfFactory extends PdfFactory<
      */
     this.emit('render:proposal', proposal, principalInvestigator, coProposers);
 
-    if (technicalReview) {
-      this.emit('render:technicalReview', proposal, technicalReview);
+    if (technicalReviews) {
```

Comment:
```suggestion
    if (technicalReviews.length > 0) {
```

Reply:
@Scott-James-Hurley I will look into this right away. Thanks for the feedback.